README.md

```diff
@@ -129,20 +129,21 @@ nationality = user_profile.get('nationality')
 
 ## Running the Examples
 
-The callback URL for both example projects will be http://localhost:5000/yoti/auth/
+The callback URL for both example projects will be ``http://localhost:5000/yoti/auth/``
 
 ### With Docker
 
-#### Flask
-
-To run the Flask container:
+To run the Flask or Django containers:
```

Comment:
Should `containers` be plural here?

Reply:
I think either works. I suppose containers could potentially imply that there were more than one container for each example, so I can change.